envi/archs/arm/regs.py

```diff
@@ -47,7 +48,7 @@
 modes.sort()
 
 reg_table = [ x for x in range(17 * REGS_PER_MODE) ]
-reg_data = [ (reg, sz) for reg,sz in arm_regs ]
+reg_data = [ (reg, sz) for reg,sz in arm_regs_tups ]
```

Comment:
What does reg_data offer that arm_regs_tups doesn't?

Reply:
farther down, reg_data is used to accumulate all the other registers (namely "banked registers") for all ARM processor modes.  this is a part of supporting per-mode registers (especially during emulation).
it's possible we could do that all with arm_regs_tups, but reg_data is dynamically generated and when i developed this i wanted to keep arm_regs_tups simple and separate.